test/playback.test.js

```diff
@@ -38,7 +38,7 @@ if (videojs.browser.IE_VERSION || videojs.browser.IS_EDGE) {
 
 QUnit.module('Playback', {
   beforeEach(assert) {
-    assert.timeout(50000);
+    assert.timeout(60000);
```

Comment:
Timeouts for playback tests need to be a bit longer when we run on github actions for some reason.